src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -145,5 +147,37 @@ public static class TimeSeriesCatalog
         public static SrCnnAnomalyEstimator DetectAnomalyBySrCnn(this TransformsCatalog catalog, string outputColumnName, string inputColumnName,
             int windowSize=64, int backAddWindowSize=5, int lookaheadWindowSize=5, int averageingWindowSize=3, int judgementWindowSize=21, double threshold=0.3)
             => new SrCnnAnomalyEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, windowSize, backAddWindowSize, lookaheadWindowSize, averageingWindowSize, judgementWindowSize, threshold, inputColumnName);
+
+        /// <summary>
+        /// Singular Spectrum Analysis (SSA) model for modeling univariate time-series.
+        /// For the details of the model, refer to http://arxiv.org/pdf/1206.6910.pdf.
+        /// </summary>
+        /// <param name="catalog">Catalog.</param>
+        /// <param name="inputColumnName">The name of the column on which forecasting needs to be performed.</param>
+        /// <param name="trainSize">The length of series from the begining used for training.</param>
+        /// <param name="seriesLength">The length of series that is kept in buffer for modeling (parameter N).</param>
+        /// <param name="windowSize">The length of the window on the series for building the trajectory matrix (parameter L).</param>
```

Comment:
>parameter L [](start = 112, length = 11)

parameter of L? Is it a symbol used in the reference paper?

Reply:
Yes, please check page 4.

---
In reply to: [289946512](https://github.com/dotnet/machinelearning/pull/1900#discussion_r289946512) [](ancestors = 289946512)